examples/eady_turbulence.jl

```diff
@@ -5,306 +5,398 @@
+# Now we're ready to animate.
+
+@info "Making an animation from saved data..."
+
+anim = @animate for (i, iter) in enumerate(iterations)
+
+    ## Load 3D fields from file
+    t = file["timeseries/t/$iter"]
+    R = file["timeseries/ζ/$iter"] ./ coriolis.f
+    δ = file["timeseries/δ/$iter"]
+
+    surface_R = R[:, :, grid.Nz]
+    surface_δ = δ[:, :, grid.Nz]
+
+    slice_R = R[:, 1, :]
+    slice_δ = δ[:, 1, :]
+
+    Rlim = 0.5 * maximum(abs, R) + 1e-9
+    δlim = 0.5 * maximum(abs, δ) + 1e-9
+
+    Rlevels = nice_divergent_levels(R, Rlim)
+    δlevels = nice_divergent_levels(δ, δlim)
+
+    @info @sprintf("Drawing frame %d from iteration %d: max(ζ̃ / f) = %.3f \n",
+                   i, iter, maximum(abs, surface_R))
+
+    R_xy = contourf(xζ, yζ, surface_R';
+                          color = :balance,
+                    aspectratio = 1,
+                         legend = false,
+                          clims = (-Rlim, Rlim),
+                         levels = Rlevels,
+                          xlims = (0, grid.Lx),
+                          ylims = (0, grid.Lx),
+                         xlabel = "x (m)",
+                         ylabel = "y (m)")
+    
+    δ_xy = contourf(xδ, yδ, surface_δ';
+                          color = :balance,
+                    aspectratio = 1,
+                         legend = false,
+                          clims = (-δlim, δlim),
+                         levels = δlevels,
+                          xlims = (0, grid.Lx),
+                          ylims = (0, grid.Lx),
+                         xlabel = "x (m)",
+                         ylabel = "y (m)")
+
+    R_xz = contourf(xζ, zζ, slice_R';
+                    aspectratio = grid.Lx / grid.Lz * 0.5,
+                          color = :balance,
+                         legend = false,
+                          clims = (-Rlim, Rlim),
+                         levels = Rlevels,
+                          xlims = (0, grid.Lx),
+                          ylims = (-grid.Lz, 0),
+                         xlabel = "x (m)",
+                         ylabel = "z (m)")
+
+    δ_xz = contourf(xδ, zδ, slice_δ';
+                    aspectratio = grid.Lx / grid.Lz * 0.5,
+                          color = :balance,
+                         legend = false,
+                          clims = (-δlim, δlim),
+                         levels = δlevels,
+                          xlims = (0, grid.Lx),
+                          ylims = (-grid.Lz, 0),
+                         xlabel = "x (m)",
+                         ylabel = "z (m)")
+
+    plot(R_xy, δ_xy, R_xz, δ_xz,
+           size = (1000, 800),
+           link = :x,
+         layout = Plots.grid(2, 2, heights=[0.5, 0.5, 0.2, 0.2]),
+          title = [@sprintf("ζ/f(t=%s) (s⁻¹)", prettytime(t)) @sprintf("δ(t=%s) (s⁻¹)", prettytime(t)) "" ""])
+
+    iter == iterations[end] && close(file)
+end
 
-makeplot!(axs, model)
-gcf()
+gif(anim, "eady_turbulence.gif", fps = 8) # hide
```

Comment:
Are we going to change all movies to gifs?

Hoping we can find a way to get mp4's to work but might be a losing battle =/

X-Ref: #944